### PR TITLE
ab: select technique page

### PR DESCRIPTION
**Pull request by annagav**

### CODE REVIEW

File: instructor/templates/instructor/select_technique.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{% extends "instructor/assignment_sidebar.html" %}
+
+{% block right_side %}
+
+    <div class='scb_s_experiment_setup_title'>Select Technique</div>
+
+    <div class='scb_s_assignment_setup_course_name_heading'>
+        Which Techniques should be available to your students?
```
Comment (noisecapella) [inline instructor/templates/instructor/select_technique.html]: Techniques should be lowercase

File: instructor/common.py [hunk 2]
```diff
@@ -286,11 +286,34 @@ def assignments_variables(request):
...
+            if 'continue' in request.POST:
+                return redirect("common_select_technique")
+
+    form = AssignmentForm(instance=assignment)
+    return render_to_response(
+        'instructor/select_technique.html',
+        {
+            'form': form,
+            'assignment_name': assignment.name,
+            'section_name': 'Select Technique',
```
Comment (noisecapella) [inline instructor/common.py:313]: To match the other pages this should be 'Experimental Techniques' instead

**Comment by noisecapella** [comment]
Looks good overall :+1:

File: instructor/common.py (hunk 2 above)
Comment (annagav) [inline instructor/common.py:313]: Alison approved this wording for now.

**Pull request merged**